docs/variants/dell_optiplex/building-manual.md

```diff
@@ -32,22 +41,100 @@ To build coreboot image, follow the steps below:
     docker run --rm -it \
        -v $PWD:/home/coreboot/coreboot \
        -w /home/coreboot/coreboot \
-       coreboot/coreboot-sdk:0ad5fbd48d /bin/bash
+       coreboot/coreboot-sdk:VERSION /bin/bash
     ```
 
+    * `VERSION` should be replaced according to version you building:
+      - `v0.1.0` - `2022-04-04_9a8d0a03db`
+
 5. Inside of the container, configure and start the build process:
 
     ```bash
-    (docker)make distclean
-    (docker)cp configs/config.dell_optiplex_9010_sff .config
-    (docker)make olddefconfig
-    (docker)make
+    make distclean
+    ```
+
+    ```bash
+    cp configs/config.dell_optiplex_9010_sff .config
+    ```
+
+    ```bash
+    make olddefconfig
+    ```
+
+    ```bash
+    make
     ```
 
     or simply:
 
     ```bash
-    (docker)make distclean && cp configs/config.dell_optiplex_9010_sff .config && make olddefconfig && make
+    make distclean && cp configs/config.dell_optiplex_9010_sff .config && make olddefconfig && make
+    ```
+
+This will produce a release binary placed in `build/coreboot.rom`. To flash
+coreboot refer to [initial deployment manual](initial-deployment.md).
+
+## Debug build
+
+1. Clone the coreboot repository:
+
+    ```bash
+    git clone https://review.coreboot.org/coreboot.git
+    ```
+
+2. Get the submodules:
+
+    ```bash
+    cd coreboot
+    ```
+
+    ```bash
+    git submodule update --init --recursive --checkout
+    ```
+
+3. Checkout Dasharo branch for OptiPlex 7010/9010 (replace vX.Y.Z with valid
+   version):
+
+    ```bash
+    git remote add dasharo https://github.com/dasharo/coreboot.git
+    ```
+
+    ```bash
+    git fetch dasharo
+    ```
+
+    ```bash
+    git checkout dell_optiplex_9010_vX.Y.Z
+    ```
+
+4. Start docker container:
+
+    ```bash
+    docker run --rm -it \
+       -v $PWD:/home/coreboot/coreboot \
+       -w /home/coreboot/coreboot \
+       coreboot/coreboot-sdk:VERSION /bin/bash
+    ```
+
+    * `VERSION` should be replaced according to version you building:
+      - `v0.1.0` - `2022-04-04_9a8d0a03db`
+
+5. Inside of the container, configure and start the build process:
+
+    ```bash
+    make distclean
+    ```
+
+    ```bash
+    cp configs/config.dell_optiplex_9010_sff .config
+    ```
+
+    ```bash
+    echo "CONFIG_DEFAULT_CONSOLE_LOGLEVEL_8=y" >> .config
```

Comment:
Default miniconfig seems to already select log level 8: https://github.com/Dasharo/coreboot/pull/202/files#diff-fca86f5811a0f493e22a50dca618b50058dd3d271447d0a06058699a6263261cR12
What about release build then? It should patch the log level to 0 I believe

Reply:
Fixed https://github.com/Dasharo/coreboot/pull/202/commits/9009e4b08a3a0fda34afc96a091cee850420d5f5

Reply:
@miczyg1 fixed https://github.com/Dasharo/docs/pull/139/commits/9b51611c21132b4ad4eb6705620899dfcb05b40c